robottelo/test.py

```diff
@@ -18,8 +18,11 @@
 from datetime import datetime
 from fabric.api import execute, settings
 from robottelo.cli.metatest import MetaCLITest
+from robottelo.cli.org import Org as OrgCli
```

Comment:
The import will require to be `import robottelo.cli.org` and then use as `robottelo.cli.org.Org`.

I think we can accept this for while and make the test module a package which will contain separated modules for ui (robottelo.test.ui), cli (robottelo.test.cli), api (robottelo.test.cli) and performance (robottelo.test.performance) and then from the package we should be able to import the testcase classes without any need of refactor.